authors/apps/notifications/utils.py

```diff
@@ -0,0 +1,30 @@
+class NotificationPermissions:
+    RECEIVE_NOTIFICATION_EMAILS = 'RECEIVE_NOTIFICATION_EMAILS'
+    FOLLOWED_AUTHOR_ARTICLE_CREATED = 'FOLLOWED_AUTHOR_ARTICLE_CREATED'
+    PROFILE_FOLLOW = 'PROFILE_FOLLOW'
+    MY_ARTICLE_COMMENTED = 'MY_ARTICLE_COMMENTED'
+    FAVORITED_ARTICLE_COMMENTED = 'FAVORITED_ARTICLE_COMMENTED'
+    MY_ARTICLE_LIKED = 'MY_ARTICLE_LIKED'
+    FAVORITED_ARTICLE_LIKED = 'FAVORITED_ARTICLE_LIKED'
+    AUTHORED_ARTICLE_COMMENTED = 'AUTHORED_ARTICLE_COMMENTED'
+    AUTHORED_ARTICLE_LIKED = 'AUTHORED_ARTICLE_LIKED'
+    AUTHORED_ARTICLE_FAVORITED = 'AUTHORED_ARTICLE_FAVORITED'
+ 
+    all = {
+        'RECEIVE_NOTIFICATION_EMAILS': 'RECEIVE_NOTIFICATION_EMAILS',
```

Comment:
@malfahad you can now refer to the class instances here inorder to have one single place with a hardcoded string e.g
```suggestion
        'RECEIVE_NOTIFICATION_EMAILS': cls.RECEIVE_NOTIFICATION_EMAILS,
```